src/objects.h

```diff
@@ -10381,6 +10373,7 @@ class Float32x4: public JSObject {
   // Helpers.
   static const int kLanes = 4;
   inline float getAt(int index);
+  inline float get_lane(int lane) {return getAt(lane);}
   inline float x() { return getAt(0); }
```

Comment:
@huningxin We use x()/y()/z()/w() many places in runtime-simd.cc including macros and functions. Are we sure to remove these functions?